website/data.yml

```diff
@@ -1,701 +1,774 @@
-# Landscape2 data
-#
-# This file contains the data that will be used to generate the landscape.
-#
-# Reference documentation: https://github.com/cncf/landscape2/blob/main/docs/config/data.yml
-
 categories:
-  - name: Inference
-    subcategories:
-      - name: Inference Engine
```

Comment:
I also updated the format of data.yaml to match the output of Python's `yaml.dump`.
So, after merging this PR, when we create a project request issue to add a new project, only the lines corresponding to the newly added items will be modified.

Reply:
Will this impact the website display? @samzong

Reply:
![CleanShot 2025-05-08 at 21 19 53@2x](https://github.com/user-attachments/assets/db45e8c6-5e05-4e8e-b71f-df5d419297c1)
I run the website locally, and it works.